.travis.yml

```diff
@@ -10,15 +10,27 @@ language: python
 
 matrix:
   include:
+    # use libgeos binary from apt
     - python: 2.6
       env: NUMPY=numpy==1.6
     - python: 2.7
     - python: 3.3
     - python: 3.4
     - python: 3.5
+    
+    # compile internal libgeos
+    - python: 2.7
+      env: BUILD_INTERNAL_GEOS=true
+    - python: 3.5
+      env: BUILD_INTERNAL_GEOS=true
```

Comment:
Personally, I would have just picked two of the existing test runs and set them to use the internal geos. It'll reduce the amount of work that Travis has to do.